packages/storybook/src/nimble/button/button.mdx

```diff
@@ -1,45 +1,269 @@
+        <NimbleButton appearance="ghost">{`Back`}</NimbleButton>
+      </div>
+      <div style={{ display: 'flex', alignItems: 'center', gap: '24px' }}>
+        <NimbleButton appearance="ghost">{`Cancel`}</NimbleButton>
+        <NimbleButton appearance="block" appearanceVariant="accent">{`Continue`}</NimbleButton>
+      </div>
+    </div>
+  </GuidelinesDo>
+  <GuidelinesDont title="Don’t Crowd the Bold" 
+    copy="Avoid placing multiple higher-impact elements in close proximity, as their competing visual weight can dilute overall effectiveness and disrupt hierarchy. Specifically don't mix Outline with Block buttons (or even Primary with Accent buttons).">
+  <div style={{ display: 'flex', flexDirection: 'column', gap: '24px', width: '100%' }}>
+      {/* Row 1: Outline + Block (don’t mix weights) */}
+      <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'space-between', width: '100%' }}>
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          <NimbleButton appearance="ghost">{`Back`}</NimbleButton>
+        </div>
+        <div style={{ display: 'flex', alignItems: 'center', gap: '24px' }}>
+          <NimbleButton appearance="outline">{`Cancel`}</NimbleButton>
+          <NimbleButton appearance="block">{`Continue`}</NimbleButton>
+        </div>
+      </div>
+
+{/* Divider to mirror Figma */}
+
+<div style={{ height: 2, width: '100%', background: 'rgba(22,22,23,0.09)' }} />
+
+      {/* Row 2: Primary Block + Accent Block (don’t mix variants) */}
+      <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'space-between', width: '100%' }}>
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          <NimbleButton appearance="ghost">{`Back`}</NimbleButton>
+        </div>
+        <div style={{ display: 'flex', alignItems: 'center', gap: '24px' }}>
+          <NimbleButton appearance="block">{`Cancel`}</NimbleButton>
+          <NimbleButton appearance="block" appearanceVariant="accent">{`Continue`}</NimbleButton>
+        </div>
+      </div>
+    </div>
+
+  </GuidelinesDont>
+</DocsGrid>
 
 ### Sizing
 
 Nimble Buttons are currently always 32px tall. Designs exist for other sizes; if
 you need these in an application, please comment on
 [Configurable height for nimble controls (#610)](https://github.com/ni/nimble/issues/610).
 
-{/* ## Usage */}
-
-{/* ## Examples */}
+{/* Spacing with Other components */}
 
 ## Accessibility
 
+### Accessible Icon Buttons
+
 <ContentHiddenDocs buttonElement="nimble-button" />
 
-{/* ## Resources */}
+### Keyboard Usage
+
+Keyboard accessibility is a critical component of our design system, ensuring
+that all users, including those with disabilities, can navigate and interact
+with our products efficiently. By adhering to best practices for keyboard usage,
+we create a more inclusive and user-friendly experience.
+
+## Alternatives & Related
+
+### Alternative to Consider
```

Comment:
I personally liked having the alternatives near the top because it quickly redirects users to the right component before they spend time reading the whole document.